trackma/utils.py

```diff
@@ -35,14 +35,45 @@
 STATUS_FINISHED = 2
 STATUS_NOTYET = 3
 STATUS_CANCELLED = 4
-STATUS_OTHER = 100
+STATUS_OTHER = -1
+
+STATUS=[ 'Unknown' , 'Airing' , 'Finished' , 'Not yet aired' , 'Cancelled' , 'Others' ]
```

Comment:
```suggestion
STATUS = ['Unknown', 'Airing', 'Finished', 'Not yet aired', 'Cancelled', 'Others']
```

Where is this used?

Reply:
We can use them to show String instead of a Number.